### PR TITLE
chore: add to-do list prototype 

### LINKED ISSUE #393: chore: add second basic css lab project to frontend cert
The Basic CSS module will have a few labs for campers to do.
Labs are defined as short projects campers build on their own. It will consist of a blank editor and user stories.
Here is an example for the format:
https://www.freecodecamp.org/learn/the-odin-project/top-build-a-recipe-project/top-build-a-recipe-project

The previous workshop will have been the cafe menu project here
https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-basic-css-by-building-a-cafe-menu/step-91

So we need a lab that covers some of those topics in that workshop.

## length of project

Even though we will have user stories for both HTML and CSS in this project, it should still be a smaller project then the cafe menu project.
So maybe 10-15 user stories?

## note about the HTML boilerplate

At this point in the cert, campers will have created a few labs and worked on a few workshops.
So we no longer need to force them to write out the HTML boilerplate each time.
So we don't need to create user stories for the HTML boilerplate anymore. 

## Note about adding new images

For adding new images, make sure they are free to use. You will also need to be add that image to our CDN repo here
https://github.com/freeCodeCamp/cdn

Once the new image has been added to the CDN repo, then you will need to reference that cdn link in your project.

If you are not sure about if an image can be used, reach out on staff chat. 

## Acceptance criteria

- [ ] create a new folder of the project in the `frontend-cert/css-projects` directory
- [ ] add your completed project in an `index.html` file and `styles.css` file in that new folder
- [ ] create a markdown file for your proposed user stories. You should call it `user-stories.md` and place it in your new project folder

### PR DESCRIPTION
**Pull request by zairahira**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #393

### CODE REVIEW

File: frontend-cert/css-projects/todo-list-stylized/user-stories.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+2. You should have an unordered list with the class `todo-list`.
+
+3. Inside the unordered list, you should have 4 list items. Each list item should have a checkbox, a task label and two links that point to a valid URL. 
+
+4. The first link in each list item should have the text `Edit` and the second link should have the text `Delete`. The links should also have the class `edit` and `delete` respectively.
+
+5. The list items should have a bottom margin of `15px` and a line height of `1.6`.
+
+6. Each list item should have squared bullets.
```
Comment (jdwilkin4) [inline frontend-cert/css-projects/todo-list-stylized/user-stories.md:11]: They haven't learned about list style type here, so we should remove this requirement

Comment (zairahira) [reply frontend-cert/css-projects/todo-list-stylized/user-stories.md:11]: In the project sheet, there is a lecture with the title "How to work with the different list-style properties" just above this lab.
I assumed that some list style types would be taught there.
Should I still remove the `list-style-type` from the prototype?

**Review by moT01 — COMMENTED** [review]
Generally, I like this @zairahira 👍 - but here's a couple notes...

All the links should have `target=_blank` I think.

It's a little strange that edit and delete buttons link to external URLs. Can we find another way to include a link with those user stories? Like maybe make the link part of the todo list item? "All of your todo items should have a link in them" maybe? If we did that, we could remove the edit and delete buttons.

It also feels a little strange to have the checkbox and the squares on the left. I would be fine with that - just mentioning it.

**Comment by zairahira** [comment]
@moT01, thanks for the feedback. I have made the following changes:

- Removed `delete` and `edit` buttons.
- Added a sub-item in the list item that should use a link.
- Removed the sqaure bullet for each main item.
- The sub-items should have a `list-style-type` of `square`.
- Links now have `target=_blank`.
- The links' `href` is currently `#`.

If these look fine, I'll update the user stories.

**Comment by moT01** [comment]
Yea, I think that looks good @zairahira 👍 

I would suggest to format your HTML and there's some comments in the CSS you could remove.

**Review by moT01 — APPROVED** [review]

**Pull request merged**